wg-releases/README.md

```diff
@@ -33,16 +32,25 @@ Save expensive human time by offloading work to inexpensive computers.
   * Listening and responding to feedback
 * Management of Heroku apps for [associated repositories](#associated-repositories)
 
+## Leadership Responsibilities
+
+In addition to responsibilities outlined in the [Charter](../charter/README.md), a Chair of the Releases Working Group should:
+
+* Run regular release audits for [supported branches](https://electronjs.org/docs/tutorial/support#supported-versions)
+* Run releases for supported branches passing the pre-set unreleased commit threshold.
```

Comment:
I think this responsibility is not exclusively the Releases WG Chair, but rather the Chair's responsibility to _ensure_ there are releases

Reply:
o yes this is subtle but very important phrasing 😅  ty, will update!